bundles/org.openhab.binding.hue/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -133,6 +153,10 @@
 		<item-type>String</item-type>
 		<label>Alert</label>
 		<description>The alert channel allows a temporary change to the bulb’s state.</description>
+		<tags>
+			<tag>Control</tag>
+			<tag>Light</tag>
```

Comment:
Actually, I think the semantic of this one should be "Alarm", not "Light". One uses it in case one want to signal an alarm and not in order to have light in the room.

Reply:
Updated to use Alarm